### PR TITLE
[#4] [UI] As a user, I can see Splash Screen 

### PR DESCRIPTION
**Pull request by ducbm051291**
#4 

## What happened

Allow user to see Splash Screen when the app launch.

image

 
## Insight

- Create SplashView and layout UI as design
- Create ImageResource extension to get Images from Asset
- Update Asset
 
## Proof Of Work

https://user-images.githubusercontent.com/24598204/201305446-5d611b8f-a08f-41d9-80db-7f2ee150f6b5.mp4

### CODE REVIEW

File: Surveys/Sources/Application/AppRouter.swift [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+//  AppRouter.swift
+//  Surveys
+//
+//  Created by David Bui on 10/11/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Combine
+
+enum Route: Hashable {
```
Comment (Shayokh144) [inline Surveys/Sources/Application/AppRouter.swift:11]: Can we take this to a separate file?

**Review by Shayokh144 — APPROVED** [review]

File: Surveys/Sources/Presentation/Modules/SplashView/SplashView.swift [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                .aspectRatio(contentMode: .fill)
+                .ignoresSafeArea()
+            Asset.logoWhiteIcon.image
+                .resizable()
+                .frame(width: 200.0, height: 48.0, alignment: .center)
+                .onAppear(perform: {
+                    withAnimation(Animation.easeInOut(duration: 1.0)) {
+                        fadeInOut.toggle()
+                    }
+                })
```
Comment (blyscuit) [inline Surveys/Sources/Presentation/Modules/SplashView/SplashView.swift:28]: I think we should use the short forms for ease of reading
```
.onAppear {
                    withAnimation(.easeIn(duration: 1.0)) {
                        fadeInOut.toggle()
                    }
                }
```

**Review by blyscuit — COMMENTED** [review]
Looks good, one comment

**Review by blyscuit — APPROVED** [review]

File: Surveys/Sources/Presentation/Modules/SplashView/SplashView.swift [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        ZStack {
+            Asset.splashScreenBackgroundImage.image
+                .resizable()
+                .aspectRatio(contentMode: .fill)
+                .ignoresSafeArea()
+            Asset.logoWhiteIcon.image
+                .resizable()
+                .frame(width: 200.0, height: 48.0, alignment: .center)
+                .onAppear {
+                    withAnimation(Animation.easeInOut(duration: 1.0)) {
```
Comment (blyscuit) [inline Surveys/Sources/Presentation/Modules/SplashView/SplashView.swift:25]: You can still short form this too
```suggestion
                    withAnimation(.easeInOut(duration: 1.0)) {
```

**Review by suho — APPROVED** [review]

**Pull request merged**